code/api/src/modules/user/query.test.js

```diff
@@ -141,21 +152,32 @@ describe('user queries', () => {
     expect(response.body.data.userSignup.email).toEqual("newEmail@example.com")
   })
 
-  it.skip ('logs in a user', async() => {
-    const loginUserQuery = `query {
-      userLogin(email: "test1@example.com", password: "abc123") {
-        name
-        email
-        role
-      },
```

Comment:
@dborski Nice job figuring out the user/admin authentication piece figured out to get your tests working! Seems challenging and like you accomplished what you set out to do.